### PR TITLE
[CI] Moved CI to GitHub actions #153

### LINKED ISSUE #153: [CI] Move CI to github actions
Move CI to github actions.

### PR DESCRIPTION
**Pull request by codesankalp**
Added ci.yml for GitHub actions.

Fixes #153

Success build URL: https://github.com/codesankalp/openwisp-wifi-login-pages/actions/runs/584957644

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  build:
+    name: Python==${{ matrix.python-version }}
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
```
Comment (devkapilbansal) [inline .github/workflows/ci.yml:21]: We need python only for qa-checks. Right?
Therefore there is no point checking for different python versions

**Review by devkapilbansal — CHANGES_REQUESTED** [review]

**Review by Purhan — COMMENTED** [review]
@codesankalp Reminder:
Don't forget to remove https://github.com/openwisp/openwisp-wifi-login-pages/blob/master/.travis.yml :smile:

**Comment by codesankalp** [comment]
@devkapilbansal @Purhan Can you review this again?
I have made the suggested changes.
And success build URL is: https://github.com/codesankalp/openwisp-wifi-login-pages/actions/runs/585779410

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up yarn
+        uses: borales/actions-yarn@v2.0.0
+        with:
+          cmd: install
```
Comment (devkapilbansal) [inline .github/workflows/ci.yml:29]: Is there any way to use cache, similar to what was done in `travis.yml` ?

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
Great work @codesankalp 
Please update the badges in README too.

Also, please look into my question below

File: .github/workflows/ci.yml (hunk 2 above)
Comment (codesankalp) [reply .github/workflows/ci.yml]: @devkapilbansal Please see again, I have used the cache this time.

**Comment by codesankalp** [comment]
**Success Build URL**: https://github.com/codesankalp/openwisp-wifi-login-pages/actions/runs/587131992
**badge with passing status**:
link: https://github.com/codesankalp/openwisp-wifi-login-pages/workflows/OpenWisp%20WiFi%20Login%20Pages%20CI%20BUILD/badge.svg?branch=master
badge: [Build Status](https://github.com/openwisp/openwisp-wifi-login-pages/actions)

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1:

**Comment by devkapilbansal** [comment]
@nemesisdesign I changed the base branch of this PR for testing purpose.

**Review by nemesisdesign — APPROVED** [review]
Thank you all @codesankalp, @Purhan, @devkapilbansal!

I will merge to the selected branch and open another PR there so we can confirm if the build passes correctly.

**Comment by nemesisdesign** [comment]
Here: https://github.com/openwisp/openwisp-wifi-login-pages/pull/174

**Pull request merged**